tests/rpc/spec/location_spec.js

```diff
@@ -45,18 +45,22 @@ describe('Location', function(){
         handleEvent('SearchResultEvent', function(data) {
             channel.log('SearchResultEvent:', data);
             expect(data.success).toBe(true);
+            expect(data.requestParameters).toEqual(searchCriteria);
             expect(data.result.locations[0].name).toContain(searchCriteria);
-            
+
             result = Object.keys(data.result.locations[0]);
-                expect(result).toContain('channelId');
-                expect(result).toContain('id');
-                expect(result).toContain('lat');
-                expect(result).toContain('lon');
-                expect(result).toContain('rank');
-                expect(result).toContain('region');
-                expect(result).toContain('type');
-                expect(result).toContain('village');
-                //expect(result).toContain('zoomScale');
+            expect(result).toContain('zoomScale');
+            expect(result).toContain('paikkatyyppi');

```

Comment:
The other keys should be ok, but `paikkatyyppi` is specific to search backend used in Paikkatietoikkuna. We could do something like add "additionalSearchResultKeys" array of values to parameter file, add `paikkatyyppi` to `paikkatietoikkuna_parameters.js` and use that here to check this.

Reply:
Added an array called additionalSearchResultKeys to paikkatietoikkuna_parameters.js. Unit test in question now loops over the entries in said array and expects the values of the array to appear in SearchResultEvent.